packages/permissions/src/components/authorized/authorized.tsx

```diff
@@ -65,21 +85,32 @@ const Authorized = (props: Props) => {
     props.actualActionRights
   );
 
-  return (
-    <React.Fragment>
-      {props.render(hasDemandedPermissions && hasDemandedActionRights)}
-    </React.Fragment>
-  );
+  const isAuthorized = Boolean(
+    props.demandedDataFences && props.demandedDataFences.length
+  )
+    ? createHasAppliedDataFence({
```

Comment:
Nitpick: can we move this to a variable like the other ones? I think it's a bit easier to read and understand the logic.

```js
const hasDemandedPermissions = /**/;
const hasDemandedActionRights = /**/;

if (Boolean(props.demandedDataFences && props.demandedDataFences.length > 0)) {
  const hasDemandedDataFences = hasAppliedDataFences(/**/);
  return <>{props.render(hasDemandedDataFences)}</>
}
return <>{props.render(hasDemandedPermissions && hasDemandedActionRights)}</>
```